doc/api/Decryption_Options.md

```diff
@@ -237,17 +237,56 @@ The `maxSessionCacheSize` option allows to configure the maximum number of
 
 _type_: `Boolean | undefined`
 
-If set to `true`, the `MediaKeySession` created for a content will be immediately closed
-when the content stops its playback.
+If set to `true`, all open `MediaKeySession` (JavaScript Objects linked to the keys used
+to decrypt the content) will be closed when the current playback stops.
 
-This might be required by your key system implementation (most often, it is not).
+By default, we keep created `MediaKeySession` from previous contents (up to
+[`maxSessionCacheSize`](#maxsessioncachesize) `MediaKeySession`) to speed-up playback and
+avoid round-trips to the license server if the user ever decide to go back to those
+contents or to other contents relying to the same keys.
 
-If set to `false` or not set, the `MediaKeySession` can be reused if the same content
-needs to be re-decrypted.
+However we found that some devices poorly handle that optimization.
 
-If you want to set this property because the current device has a limited number of
-`MediaKeySession` that can be created at the same time, prefer using `maxSessionCacheSize`
-instead.
+Note that if setting that property to `true` fixes your issue, it may be that it's just
+the [`maxSessionCacheSize`](#maxsessioncachesize) which is for now too high.
+
+However if your problem doesn't disappear after setting `closeSessionsOnStop` to `true`,
+you may try [`reuseMediaKeys`](#renewmediakeys) next.
+
+### reuseMediaKeys
+
+If set to `true` or if not set, we might rely on the previous `MediaKeys` if a compatible
+one is already set on the media element, allowing to potentially speed-up content
+playback.
+
+If set tot `false`, we will create a new `MediaKeys` instance (a JavaScript object needed
+to decrypt contents) if needed for that content.
+
+We noticed that reusing a previous `MediaKeys` had led to errors on a few devices. For
+example some smart TVs had shown errors after playing several encrypted contents, errors
+which disappeared if we renewed the `MediaKeys` for each content.
```

Comment:
I'm wondering if those details are relevant in the documentation

Reply:
Why wouldn't it?

To me it explain why this option exists and why it may be useful no?